Sources/_SwiftFormatTestSupport/MarkedText.swift

```diff
@@ -18,23 +21,37 @@ public struct MarkedText {
   /// The text with all markers removed.
   public let textWithoutMarkers: String
 
+  /// If the marked text contains "➡️" and "⬅️", they're used to create a selection
+  public var selection: Selection?
+
```

Comment:
Since all the markers are represented as `Int` that signify UTF-8 offsets, I think it would make sense to also store the ranges as `Range<Int>`.

Reply:
I'm now using `Range<Int>` as they're built up, but create a `Selection` in the initializer. I may want to revisit this (and not have a `selection` property) if we support line ranges.